namespace csharp Ruyi.SDK.Constants
namespace cpp Ruyi.SDK.Constants

const i32 low_latency_socket_port = 11290

const i32 high_latency_socket_port = 11390

const string layer0_broker_address = "inproc://{addr}:5555"

const string layer0_publisher_in_uri = "tcp://{addr}:5567"

const string layer0_publisher_out_uri = "tcp://{addr}:5568"

const string setting_config_folder = "resources/configs/"

const string setting_system_config = "resources/configs/systemsetting"

const string setting_system_user_config = "resources/configs/usersetting"

const string system_setting_version = "1.0.0.1"

const string layer0_debugger_channel = "layer0_debugger_channel"

const string trc_test_channel = "trc_test_channel"

const string debugger_publisher_in_uri = "tcp://{addr}:8867"

const string debugger_publisher_out_uri = "tcp://{addr}:8868"

const string broker_playback_message = "mmi.developer.playback"

const string broker_power_message = "mmi.power.operation"


